openerp/models.py

```diff
@@ -5315,6 +5319,20 @@ def sudo(self, user=SUPERUSER_ID):
 
         Returns a new version of this recordset attached to the provided
         user.
+
+        By default this returns a `SUPERUSER` recordset, where access control
+        and record rules are bypassed.
+
+        .. warning::
+            A recordset for a different user may also be attached to a
+            different company. As a consequence, and depending on your
+            application, using `sudo` may break features in multi-company
+            instances.
```

Comment:
Here as well, I would not write that this may *break features*. Factually, `sudo` could cause data access to cross the boundaries of record rules, possibly mixing records that are meant to be isolated (e.g. records from different companies in multi-company environments).